packages/apib-parser/lib/adapter.js

```diff
@@ -42,9 +42,29 @@ function parse({
     requireBlueprintName,
   };
 
-  return drafter.parse(source, options);
+  return drafter.parse(source, options).then((result) => {
+    const parseResult = namespace.fromRefract(result);
+    const isAnnotation = element => element.element === 'annotation';
+    const { Link } = namespace.elements;
+
+    if (!isAnnotation(parseResult.content[0])) {
```

Comment:
Can you please explain the intent of this check? I am not sure it is needed.

Reply:
In case of error it does not add the format link, do you want to remove it?

Reply:
I think in all cases we should include the link element, some editors such as Apiary show the source format on annotations which the link element's title will be useful for:

![Screenshot 2020-08-17 at 17 03 48](https://user-images.githubusercontent.com/44164/90417511-b6da2300-e0ab-11ea-8cc7-3ae2e33c8245.png)

The use-case may be finding the underlying format which caused the error. The "Unknown" here would become "OpenAPI" etc.

Reply:
removed condition https://github.com/apiaryio/api-elements.js/pull/534/commits/e310ea38b517a25339e50e8b3c62679204790438